CHANGES.rst

```diff
@@ -16,6 +16,7 @@ releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_
   date (:ghuser:`m-pannier`, :ghuser:`lillyfischer`).
 * :gh:`393` normal retirement age adjustment aligned with the rules.
   (:ghuser:`TeBackh`).
+* :gh:`398` Implement pension for women (:ghuser:`TeBackh`).
```

Comment:
#398 and this PR are a bit mixed together. Two options:

- separate them such that at least one of the PRs does not contain the other and can be merged first
- just make one big PR out of them and close the other one

Reply:
I merged the branch 397-implementation-of-pension-for-women into this branch to update variable names etc. I will merge again, when the PR 398 is closed.

Reply:
branch 397 is not in PR any more. now both issues linked to this PR